calcom/cal.com
# Team webhooks

## What does this PR do?

- Adds webhooks for teams. Only admins and owners can create, edit and delete webhooks:

Admin/Owner: 
<img width="869" alt="Screenshot 2023-05-16 at 14 19 47" src="https://github.com/calcom/cal.com/assets/30310907/ec0b64cd-f3a3-4070-8abf-dbbf745d36ae">

Member: 
<img width="835" alt="Screenshot 2023-05-16 at 14 20 06" src="https://github.com/calcom/cal.com/assets/30310907/2c7fc54b-4277-4adf-b415-643e04f510d5">

- Also adds event type webhooks for user event types. Until now only team event types could have even type webhooks

Fixes #8815

## Type of change

- New feature (non-breaking change which adds functionality)
- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- Create team, event type and user webhooks 
- Test editing and deleting webhooks
- Test with users that have different membership roles 
- Test if webhooks trigger as expected
